### DISCUSSION TITLE
Touch panel o tablet da muro per HA

### DISCUSSION THREAD

**Discussion by Salsalove** [indomus/forum#1205] [Hardware & Firmware]
Salve ragazzi, sto ristrutturando casa e vorrei installare un touch panel o un tablet al muro per gestire la demotica con HA. 
Quale prodotto mi consigliate? 
Nel caso di tablet, l'unico mio dubbio è che la maggior parte ha l'attacco per la ricarica sui bordi e quindi si vedrebbe una volta installato a muro. 

Attendo consigli...grazie! ;)

**Comment by dante70** [comment]
ciao,
io ho optato per tablet economici  pero' full hd. uno dei 2 lo ho ancorato a muro attraverso dei fori fatti sulla cover rigida.
ho spessorato dietro con un foglio di plastica di qualche mm nel quale ho applicato un intaglio per far passare il cavo di alimentazione.
Si il cavo si vede ma se prendi cavi con connettore usb  secondo me non e' brutto da vedere ovviamente dello stesso colore del tablet(nel mio caso nero).Avevo nel posto dove ho installato un scatola 503 con possibilita' di far entrare l'alimentazione.
il secondo non lo ho ancora posizionato.
Sono dei Teclast e per ora vanno bene

**Reply by dante70** [reply]
usb a 90 gradi

**Reply by riccionese** [reply]
... o a 180°

![image](https://user-images.githubusercontent.com/62329292/146634423-6abecde4-c43d-493e-97c9-7abdf15f0280.png)

**Comment by Salsalove** [comment]
Grazie per io consiglio…però ho un altro dubbio: questi tablet non sono stati progettati per essere perennemente in carica e perennemente accesi…o sbaglio? Potrebbero esplodere le batterie?

**Reply by riccionese** [reply]
in un altro post, che ora non ritrovo, qualcuno aveva aggiunto uno shelly per gestire l'alimentazione .... dell'alimentatore del tablet.
Tramite i sensori che possono leggere lo stato della batteria del tablet, a 80% staccava e riattaccava quando scendeva sotto il 20%

**Reply by Salsalove** [reply]
Questo potrebbe essere interessante…👍🏼😊

**Comment by dante70** [comment]
confermo quanto scritto sopra.....io ho messo un sonoff mini che a da tensione all'alimentatore del tablet quando serve.
Esponi in HA il livello di carica del tablet e poi con automzione fai i cicli di carica al bisogno.
Se cerchi ci sono tante persone che l'hanno fatto (io ho scopiazzato dal web e ho adattato alle mie esigenze)

**Comment by Salsalove** [comment]
Grazie ancora per i consigli 😉

**Comment by giovannicaliendo** [comment]
Ciao, io come chiosco utilizzo un Raspberry 3B con il suo touch screen da 7 pollici. Non ha il problema della batteria poiché è alimentato solo dalla corrente di casa tramite il suo alimentatore (da almeno 3A). E' più macchinoso da preparare perché devi prima installare il sistema operativo e diversi packages, poi devi intervenire con l'editor su alcuni file di configurazione per far sì che parta in automatico Chromium direttamente sulla pagina di HA. Se ti interessa questa soluzione, ho creato un semplice howto per fare il setup.
Ciao
